brewpi-script/brewpi.py

```diff
@@ -753,6 +751,11 @@ def trigger_refresh(read_values=False):
 
                         newRow = prevTempJson
 
+                        # Moved this so that the last read values is updated even if logging is off. Otherwise the getDashInfo 
```

Comment:
Good catch!

Reply:
I guess this change will affect all the push targets, so it would be a good thing to test all of them so there is no strange side effects. I just tested the brewfather push but can test the grainfather as well